scripts/local/flake8_plugin.py

```diff
@@ -0,0 +1,177 @@
+import ast
```

Comment:
I put this in `scripts/local/` to avoid `src/sentry/__init__.py` (which adds 180ms to startup)